src/chocolatey.tests.integration/scenarios/InfoScenarios.cs

```diff
@@ -0,0 +1,315 @@
+        }
+
+        [ConcernFor("info")]
+        public abstract class CommandScenariosBase : TinySpec
+        {
+            protected ChocolateyConfiguration Configuration;
+            protected ICommand Command;
+
+            public override void Context()
+            {
+                Configuration = Scenario.info();
+                Scenario.reset(Configuration);
+
+                Command = NUnitSetup.Container.GetAllInstances<ICommand>()
+                    .Where(c => c.GetType() == typeof(ChocolateyInfoCommand)).First();
+            }
+
+            public override void Because()
+            {
+                MockLogger.reset();
+
+                Command.run(Configuration);
+            }
+        }
+
+        [Broken, Pending("Need to be fixed in either NuGet.Client or before calling code in NuGet.Client")]
+        public class when_searching_for_exact_package_through_command : CommandScenariosBase
+        {
+            public override void Context()
+            {
+                base.Context();
+
+                Configuration.PackageNames = Configuration.Input = "installpackage";
+
+                Configuration.Sources = "PackageOutput";
+                Scenario.add_packages_to_source_location(Configuration, "installpackage.*" + NuGetConstants.PackageExtension);
+            }
+
+            [Fact]
+            public void should_log_standalone_header_with_package_name_and_version()
+            {
+                MockLogger.Messages.Keys.ShouldContain(LogLevel.Info.to_string());
+                MockLogger.Messages[LogLevel.Info.to_string()].ShouldContain("installpackage 1.0.0");
+            }
+
+            [Fact]
+            public void should_log_package_information()
+            {
+                var lastWriteDate = File.GetLastWriteTimeUtc(Path.Combine("PackageOutput", "installpackage.1.0.0" + NuGetConstants.PackageExtension))
+                    .ToShortDateString();
+
+                MockLogger.Messages.Keys.ShouldContain(LogLevel.Info.to_string());
+                MockLogger.Messages[LogLevel.Info.to_string()].ShouldContain(" Title: installpackage | Published: 14.12.2022\r\n Number of Downloads: n/a | Downloads for this version: n/a\r\n Package url\r\n Chocolatey Package Source: n/a\r\n Tags: installpackage admin\r\n Software Site: n/a\r\n Software License: n/a\r\n Summary: __REPLACE__\r\n Description: __REPLACE__\r\n".format_with(lastWriteDate));
+            }
+
+            [Fact]
+            public void should_log_package_count_as_warning()
+            {
+                MockLogger.Messages.Keys.ShouldContain(LogLevel.Warn.to_string());
+                MockLogger.Messages[LogLevel.Warn.to_string()].ShouldContain("1 packages found.");
+            }
+        }
+
+        [Broken, Pending("Need to be fixed in either NuGet.Client or before calling code in NuGet.Client")]
+        public class when_searching_for_exact_package_with_dot_source : when_searching_for_exact_package_through_command

```

Comment:
Since "dot source" has a different meaning in PowerShell, it might be good to rename this to something else.

Perhaps `when_searching_for_exact_package_with_relative_path_dot_source` ?